src/Websites/Websites.Test/ScenarioTests/CertificatesTests.ps1

```diff
@@ -0,0 +1,141 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests creating a new managed cert for app.
+#>
+function Test-NewAzWebAppCertificate
+{
+	$rgname = "RG-W-CUS"
+	$appname = "managedcerts"
+	$slot = "testslot"
+	$prodHostname = "www.managedcerts.net"	
```

Comment:
Is it possible to use the same domain we use for the other tests so we don't have to create a new domain for this test & the SSL binding test. Thanks!

Reply:
modified to use the same domains  used for SSL binding tests